### DISCUSSION TITLE
User count mismatch on course dashboard and reports. would like to how it works

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#712] [General Discussion]
Hello all,
We are using the 5.0 version of Sunbird
The user count on the course dashboard and the user count in reports are mismatched.
I want to how and why it is happening
Also, want to know how the user count data are fetched on the dashboard and in the report.
Want to know the workaround for user count. 
If anyone has any idea regarding it please guide me as I need some assistance here
Sharing some screenshot of the dashboard and report for reference 
![image (27)](https://github.com/Sunbird-Ed/Community/assets/109504832/c132df60-e1fa-4514-8164-511d9a6db462)
![image (28)](https://github.com/Sunbird-Ed/Community/assets/109504832/f35e744d-79bd-4939-a050-6889f19370e8)
@vinukumar-vs @rajeevsathish @rajeshkumaravel @Krishnaj20

**Comment by anisha147** [comment]
Hi @sachinb80 

This query belongs to the Lern building block. Request you to raise this query in [Lern's](https://github.com/orgs/Sunbird-lern/discussions) discussion forum.

Thanks!

**Reply by sachinb80** [reply]
@anisha147 Sure

**Comment by HarishGangula** [comment]
As we haven't received a response from you, we will proceed to close this thread. If you have any further questions or require assistance in the future, please don't hesitate to start a new thread. Thank you.
